src/components/Inputs/PasswordInput.js

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react';
 import { Text, View, TextInput, StyleSheet, TouchableOpacity, SafeAreaView, KeyboardAvoidingView, Keyboard} from 'react-native';
-import { Ionicons } from '@expo/vector-icons';
 import { TouchableWithoutFeedback } from 'react-native-gesture-handler';
+import EyeIcon from '../../icons/Eye';
+import EyeClosedIcon from '../../icons/EyeClosed';
 
 const PasswordInput = () => {
```

Comment:
This component has no way to call back into the screen or component that is using it. It needs props.